__tests__/healthcareProfessional.test.ts

```diff
@@ -0,0 +1,211 @@
+import { expect } from '@jest/globals'
+import resolvers from '../src/resolvers'
+import loadSchema from '../src/schema'
+import request from 'supertest'
+import { ApolloServer } from '@apollo/server'
+import { startStandaloneServer } from '@apollo/server/standalone'
+import { initiatilizeFirebaseInstance } from '../src/firebaseDb'
+import { ErrorCode } from '../src/result'
+
+const facilityIds = [] as string[]
+
+const facilityQueryData = {
+    query: `mutation CreateFacilityWithHealthcareProfessional($input: FacilityInput) {
+        createFacilityWithHealthcareProfessional(input: $input) {
+          id
+        }
+      }`,
+    variables: {
+        input: {
+            contact: {
+                address: {
+                    addressLine1En: null,
+                    addressLine1Ja: null,
+                    addressLine2En: null,
+                    addressLine2Ja: null,
+                    cityEn: null,
+                    postalCode: null,
+                    cityJa: null,
+                    prefectureEn: null,
+                    prefectureJa: null
+                },
+                email: null,
+                mapsLink: null,
+                phone: null,
+                website: null
+            },
+            healthcareProfessionalIds: null,
+            healthcareProfessionals: [
+                {
+                    acceptedInsurance: 'JAPANESE_HEALTH_INSURANCE',
+                    degrees: [
+                        {
+                            nameJa: null,
+                            nameEn: null,
+                            abbreviation: null
+                        }
+                    ],
+                    facilityIds: [],
+                    names: [
+                        {
+                            middleName: null,
+                            locale: 'ENGLISH',
+                            lastName: 'null',
+                            firstName: 'null'
+                        }
+                    ],
+                    specialties: [
+                        {
+                            names: [
+                                {
+                                    name: null,
+                                    locale: null
+                                }
+                            ]
+                        }
+                    ],
+                    spokenLanguages: [
+                        {
+                            nameNative: 'null',
+                            nameEn: 'null',
+                            iso639_3: 'null',
+                            nameJa: 'null'
+                        }
+                    ]
+                }
+            ],
+            isDeleted: null,
+            nameEn: null,
+            nameJa: null
+        }
+    }
+
+}
+
+const healthcareProfessionalQueryData = {
```

Comment:
nit: we should be more specific with what type of query this is. Passing, failing, with languages, etc.

Reply:
Since we don't have any examples of that in the repo, please use the `suggestion` button to suggest the specific code change you want and I'll make sure the other tests get updated as well. Otherwise, we could end up going back and forth because I'm not exactly sure what you're asking for.